ui/desktop/app/components/hidden-secret/index.hbs

```diff
@@ -1,9 +1,20 @@
 <div class='hidden-secret'>
+  <Rose::Icon
+    @name={{if this.secretVisible 'visibility-show' 'visibility-hide'}}
+    @size='large'
+    {{on 'click' this.toggleSecretVisibility}}
+  />
   <Copyable
     @text={{@secret}}
     @buttonText={{t 'actions.copy-to-clipboard'}}
     @acknowledgeText={{t 'states.copied'}}
   >
-    <span>{{this.textMask}}</span>
+    <span>
```

Comment:
Not sure if this `span` wrapper is needed as copyable component adds a span wrapper with class `copyable-content`.

Reply:
True!